framework/src/materials/ParsedMaterialBase.C

```diff
@@ -76,4 +88,84 @@ ParsedMaterialBase::ParsedMaterialBase(const InputParameters & parameters)
   // get tolerance vectors
   _tol_names = parameters.get<std::vector<std::string>>("tol_names");
   _tol_values = parameters.get<std::vector<Real>>("tol_values");
+
+  // get functor vectors
+  _functor_names = parameters.get<std::vector<MooseFunctorName>>("functor_names");
+  _functor_symbols = parameters.get<std::vector<std::string>>("functor_symbols");
+
+  // validate all vector names (constants, tolerances, and functors)
+  validateVectorNames();
+}
+
+void
+ParsedMaterialBase::validateVectorNames(const std::set<std::string> reserved_names)
+{
+  // helper method to raise an paramError
+  auto throwParamErr = [this](std::string param_name, std::string msg)
```

Comment:
the default in moose for `paramError/mooseError` is to abort instead of throwing so the name is misleading unless you've toggled the global `Moose::_throw_on_error`

Reply:
I changed it to "raiseErr"